rotkehlchen/db/reports.py

```diff
@@ -302,12 +331,15 @@ def get_report_data(
             query = f'SELECT COUNT(*) FROM pnl_events {query}'
             results = cursor.execute(query, bindings).fetchone()
             total_filter_count = results[0]
+            entries_found = len(records)
         else:
             total_filter_count = len(records)
+            entries_found = total_filter_count
```

Comment:
entries found in both cases is len(records) right? you can define it outside the if/else and then in this else branch total_filter_count = entries_found